addons/mail/static/src/core_ui/message.js

```diff
@@ -253,6 +253,15 @@ export class Message extends Component {
         return this.hasOpenChatFeature && !this.message.isSelfAuthored;
     }
 
+    get hasSomeFilePreviewsVisible() {
+        return Boolean(
+            this.message.attachments.length > 0 ||
+                (this.message.linkPreviews.length > 0 &&
+                    this.store.hasLinkPreviewFeature &&
+                    !this.message.linkPreviewSquash)
+        );
+    }
+
```

Comment:
This is not used?

Reply:
Indeed, residuals of attempt to conditional based on shown attachments. I did the opposite instead with conditional on presence of text :)